google/cloud/spanner_v1/transaction.py

```diff
@@ -423,7 +423,7 @@ def execute_update(
                 response = self._execute_request(
                     method,
                     request,
-                    "CloudSpanner.ReadWriteTransaction",
+                    "CloudSpanner.execute_update",
```

Comment:
```suggestion
                    "CloudSpanner.execute_sql",
```

Reply:
But the method is in transaction.execute_update, why .execute_sql?